.lando.base.yml

```diff
@@ -14,9 +14,18 @@ env_file:
   - .env
 services:
   appserver:
+    scanner: false
     xdebug: debug
     build_as_root:
       - /app/.lando/scripts/appserver_build.sh
+    overrides:
+      environment:
+        # Support debugging CLI with XDEBUG.
+        # Ensure PHPStorm->Preferences->PHP->Servers->Name is set to appserver.
+        PHP_IDE_CONFIG: "serverName=appserver"
```

Comment:
Shamelessly borrowed from DEPT. Might not work here across all sites but setting here means less fiddling with a custom phpunit.xml. The latter _could_ be easier if you're waiting to run phpunit across multiple sites... or ideally we could give it a command line parameter. The sqllite db is important though as it's much faster than a mysql equivalent